src/services/azblob/dir_stream.rs

```diff
@@ -119,11 +122,15 @@ impl futures::Stream for DirStream {
                         );
 
                         debug!(
-                            "dir object {} got entry, mode: {}, path: {}",
-                            &self.path,
-                            de.mode(),
-                            de.path()
-                        );
+                        "dir object {} got entry, mode: {}, path: {}, content length: {:?}, last modified: {:?}, content_md5: {:?}, etag: {:?}",
```

Comment:
I prefer to keep the same. We will migrate that logic to `LoggingLayer`.

Reply:
Before migrating, Logging all fields is ok I think.